DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Administrator/CentreAdministratorsViewModel.cs

```diff
@@ -31,17 +31,15 @@ public class CentreAdministratorsViewModel : BaseSearchablePageViewModel
             IEnumerable<AdminUser> adminUsers,
             IEnumerable<string> categories,
             string? searchString,
-            string sortBy,
-            string sortDirection,
             string? filterString,
             int page
-        ) : base(searchString, sortBy, sortDirection, page, filterString)
+        ) : base(searchString, page, false, true, filterString: filterString)
         {
             CentreId = centreId;
             var sortedItems = GenericSortingHelper.SortAllItems(
                 adminUsers.AsQueryable(),
-                sortBy,
-                sortDirection
+                DefaultSortByOptions.Name.PropertyName,
+                Ascending

```

Comment:
I don't think it really matters because of Linq. They were in the same order for the courses etc. pre-admin search ticket. I've just kept the order of calls and stuck in a filter after the search.